dbtvault-dev/macros/tables/bridge.sql

```diff
@@ -254,19 +254,19 @@ candidate_rows AS (
 
 bridge AS (
     SELECT
-        {{ src_pk }},
-        AS_OF_DATE
+        {{ dbtvault.prefix([src_pk], 'c') }},
+        c.AS_OF_DATE
         {%- for bridge_step in bridge_walk.keys() -%}
         {% set bridge_link_pk = bridge_walk[bridge_step]['bridge_link_pk'] -%}
-        {{ ',' ~ bridge_link_pk }}
+        {{ ',c.' ~ bridge_link_pk }}
         {%- endfor %}
-    FROM candidate_rows
+    FROM candidate_rows c
```

Comment:
We need to be explicit with aliases. "AS c"

Reply:
I usually use the explicit syntax, but this one has escaped my attention.

Reply:
There are a few others I noticed, I didn't think it made sense to comment on all of them. Please have a look through

Reply:
You should be able to use a regex to find them easily with PyCharm.

Search:
FROM ([a-z]+) ([a-z]{1,2})

Replace:

FROM $1 AS $2